### DISCUSSION TITLE
Deepmd lammps error!

### DISCUSSION THREAD

**Discussion by dlProjectYue** [deepmodeling/deepmd-kit#4448] [Q&A]
I have trained a dp model and want to validate the results by using lammps to run MD. But when I use an origin script without no errors to run MD, it shows the errors bellow:
![errors_lmp](https://github.com/user-attachments/assets/927eccaf-6e4b-413d-b7c3-e0bf82321f58)

The script I use is shown bellow:
compute cchunk all chunk/atom bin/1d z lower ${fraction} units reduced
compute stress all stress/atom NULL
variable press atom -(c_stress[1]+c_stress[2]+c_stress[3])/(3.0*vol*${fraction})
compute binpress all reduce/chunk cchunk sum v_press
PLEASE HELP ME !

**Comment by njzjz** [comment]
Please follow the error message.

> variable press atom -(c_stress[1]+c_stress[2]+c_stress[3])/(3.0_vol_${fraction})

I don't quite understand here why only the diagonal elements of the stress are used.
